README.md

```diff
@@ -36,150 +49,158 @@ terraform init -upgrade
+    lease_days                 = 15                           //number of lease days
+
+    deployment_configuration = {
+        "BPCustomProp"               = "custom depl prop"     //custom property in BP required while requesting a catalog item
+        "Container"                  = "App.Container"        //property of a property group
+        "Container.Auth.User"        = "var.container_user"   //property of a property group
+        "Container.Auth.Password"    = "var.container_pw"     //property of a property group
+        "Container.Connection.Port"  = "var.container_port"   //property of a property group
+        "businessGroups" = <<EOF                              //this is an example to property of type array of strings
+        [
+            "bgTest1",
+            "bgTest2"
+        ]
+        EOF
+    }
 
-```
+    resource_configuration {
+        component_name              = "Windows"       //This is the component name and need not be prefixed with all properties
+        cluster                     = 2               //cluster is added as a property in the schema. Modifying it will do the
+                                                      // Scale In/Out Day 2 actions
+        configuration = {
+            cpu                     = 2
+            memory                  = 1024
+            vm_custom_prop          = "a custom prop"
+        }
+    }
 
-Save this configuration in `main.tf` in a path where the binary is placed.
+    resource_configuration {
+        component_name              = "Linux"      //This is the component name and need not be prefixed with all properties
+        configuration = {
+            cpu                     = 2
+            security_tag = <<EOF                   //this is an example to property of type array of strings
+            [
+                "dev_sg",
+                "prod_sg"
+            ]
+            EOF
+        }
+    }
 
-### Deployment Destroy
-In case you are encountering the following error in vRA7:
-```
-Insufficient entitlement to destroy VM
+    resource_configuration {
+        component_name              = "http"      //This is the component name and need not be prefixed with all properties
+        configuration = {
+            hostname                = "xyz.com"          //HTTP (apache) hostname
+            network_mode            = "bridge"           //HTTP (apache) network mode
+        }
+    }
+}
 ```
-Please set `deployment_destroy = true` which will cause the provider to destroy the deployment resource instead.
-
-### Nested structures
 
-At the moment Terraform SDK does not support nested dynamic types, which are used by vRA API.
-Issue tracking development of this feature is on GitHub (Support the pseudo dynamic type)[https://github.com/hashicorp/terraform-plugin-sdk/issues/248].
+* Import functionality is now supported for the vra7_deployment resource. If there is an exiting deployment, it can be imported by catalog item request id.
```

Comment:
Might need to highlight this section better and deemphasize the "# main.tf" below..."### main.tf" instead?